examples/sst2/requirements.txt

```diff
@@ -0,0 +1,7 @@
+absl
+clu
+flax
+jax
+ml_collections
+https://github.com/tensorflow/datasets/archive/master.zip
```

Comment:
It's because there is an issue with glue/sst2 in the latest official version. It's been fixed on github and this was the only way to get that through pip.

Reply:
Ok that makes sense! Maybe add a comment explaining that and add the current date? Then we can later change this to `tensorflow-datasets` when they release a new version on pypi.

Reply:
Ok that makes sense! Could you add a comment explaining that and add the current date? Then we can later change this to `tensorflow-datasets` when they release a new version on pypi.

Reply:
Added a comment